### PR TITLE
Set up automated publishing

### PR DESCRIPTION
**Pull request by eed3si9n**
Fixes https://github.com/sbt/sbt-github-actions/issues/132

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -51,3 +61,27 @@ enablePlugins(SbtPlugin)
 
 scriptedLaunchOpts ++= Seq("-Dplugin.version=" + version.value)
 scriptedBufferLog := true
+
+ThisBuild / homepage := Some(url("https://github.com/djspiewak/sbt-github-actions"))
```
Comment (armanbilge) [inline build.sbt:65]:
```suggestion
ThisBuild / homepage := Some(url("https://github.com/sbt/sbt-github-actions"))
```

**Review by mdedetrich — APPROVED** [review]
LGTM

**Comment by mdedetrich** [comment]
Yes we haven't made an actual release for the new org yet, I got a bit sick this week which is when I wanted to look into it (there are still various open points about what should be in the next release and whether it should be an epic bump or not).

**Comment by mdedetrich** [comment]
This is the details for the latest version

```
addSbtPlugin("com.codecommit" % "sbt-github-actions" % "0.14.2")
```

**Comment by mzuehlke** [comment]
Thanks, yes wasn't too hard to find the working details

**Pull request merged**